packages/rocketchat-geoip-plugin/server/server.js

```diff
@@ -0,0 +1,37 @@
+Accounts.onLogin(function(user) {
+	if (!user.user._id || !user.user.username) {
+		return;
+	}
+	const apiUrl = 'http://ip-api.com/json';
```

Comment:
Is there any limit on its use? Maybe use some local gis database you can download and add as a package?